pkg/kube/client.go

```diff
@@ -187,6 +187,9 @@ type CLIClient interface {
 	// ApplyYAMLFiles applies the resources in the given YAML files.
 	ApplyYAMLFiles(namespace string, yamlFiles ...string) error
 
+	// ApplyYAMLFiles applies the resources in the given YAML strings.
+	ApplyYAMLStrings(namespace string, yamls ...string) error
```

Comment:
comment is not right

Reply:
Maybe `ApplyYAMLContents` offers better clarity here